### ISSUE TITLE
How do install orion version from github ?

### ISSUE THREAD

**Issue by woowoots** [alamminsalo/orion#119]
I'am sorry, but i dont understand at all how to install the github version of Orion

There is only the 
git clone https://github.com/alamminsalo/orion
cd orion
mkdir build && cd build
qmake ../
make && sudo make install

But ? where do i use thoses command ? ( yes i'am a big noob when it come to thoses things and so far i didnt found and "simple how to"

because here 
https://alamminsalo.github.io/orion/

Look like the Orion version is outdated since the twitch chat still not have the emote / pseudo color / more space between answer and so on.

Thanks !

**Comment by alamminsalo** [comment]
It's a deep rabbit hole if you're not used to build software from source :) What platform are you on? I think just updating the installers would be easier for both

**Comment by XenonPK** [comment]
A new release is definitely in order, the amount of changes since 1.4 (feature-wise) are more than enough to warrant it, in my opinion.

Also, windows people don't have a good & easy way to get the new features, unless they have the build environment setup and know what they are doing.

**Comment by ahjolinna** [comment]
I agree that a new version is indeed needed at least for windows and mac user ...as its really pain to build on those platforms...
anyway as there has been so many changes since v1.4 maybe it would be good to release a Beta or RC version first to test if there are any new problems, and as there has been problems with some mac/win builds in the past it would be good to see if they have/can be fixed.

**Comment by woowoots** [comment]
I'am on windows ^^ 
Sorry, needed to ask since you dont seem to update the installer that often :) 

And thanks ;)

**Comment by alamminsalo** [comment]
Alright. I'll draft a new win-release later tonight, and put it up as RC-release then

**Comment by woowoots** [comment]
thanks that's cool :)

**Comment by alamminsalo** [comment]
orion-1.5.1 windows installer can now be found in releases section

**Comment by woowoots** [comment]
look like i'am really blind, but where is that "release section" ? oO
I only know that link https://alamminsalo.github.io/orion/

**Comment by ahjolinna** [comment]
@alamminsalo : this reminded me, have you checked out [Qt Installer Framework](http://doc.qt.io/qtinstallerframework/ifw-overview.html)? it has online installer/updater function and it works on all 3 platforms...would be at least useful for mac and windows users. Maybe there is another better solution for "self-updating" that can be used, I don't know what other opensource (Qt) project uses on windows and mac but this is at least Qt's own solution, maybe @rakslice knows as you are windows user.

manual : http://doc.qt.io/qtinstallerframework/
wiki: https://wiki.qt.io/Qt-Installer-Framework

**Comment by alamminsalo** [comment]
Yes, I've checked it previously, but I think the simplest updater implementation would be to just compare tags with github and show a dialog that a newer version is available, with link to open in browser.

**Comment by ahjolinna** [comment]
@alamminsalo : sure a just a simple update notification system is fine...better than nothing
